src/accountsdb/index.zig

```diff
@@ -1,18 +1,16 @@
 //! all index related structs (account ref, simd hashmap, …)
-
 const std = @import("std");
-const lib = @import("../sig.zig");
-
-const Slot = lib.core.time.Slot;
-const Pubkey = lib.core.pubkey.Pubkey;
-const FileId = lib.accounts_db.accounts_file.FileId;
-const RwMux = lib.sync.RwMux;
-
-const swiss_map = @import("swiss_map.zig");
-pub const SwissMapManaged = swiss_map.SwissMapManaged;
-pub const SwissMapUnmanaged = swiss_map.SwissMapUnmanaged;
-pub const BenchmarkSwissMap = swiss_map.BenchmarkSwissMap;
-pub const BenchHashMap = swiss_map.BenchHashMap;
+const sig = @import("../sig.zig");
+
+const Slot = sig.core.time.Slot;
+const Pubkey = sig.core.pubkey.Pubkey;
+const FileId = sig.accounts_db.accounts_file.FileId;
+const RwMux = sig.sync.RwMux;
+
+pub const SwissMapManaged = sig.accounts_db.swiss_map.SwissMapManaged;
+pub const SwissMapUnmanaged = sig.accounts_db.swiss_map.SwissMapUnmanaged;
+pub const BenchHashMap = sig.accounts_db.swiss_map.BenchHashMap;
+pub const BenchmarkSwissMap = sig.accounts_db.swiss_map.BenchmarkSwissMap;
```

Comment:
I recently started using this approach and I'm curious what you think:

```
const sig = @import("../sig.zig");
const accountsdb = @import("lib.zig");

const RwMux = sig.sync.RwMux
const SwissMapManaged = accounts_db.swiss_map.SwissMapManaged;
```

I think of each folder as a self-contained package. When packages import from other packages, they go through sig. But when importing from other files within the same package, you don't necessarly need to break out of that scope and make the package look for itself within sig as a whole. It feels weird to depend on the global scope as an intermediary, just for a regional scope to import from itself.

Reply:
i dont have a big preference since they seem very similar and its a relatively small thing

i like the global scopes because its a consistent import pattern and you dont need to understand the how 'lib.zig' relates to imports to understand whats going on

i agree it is strange to depend on the global scope when regional works so im fine if we take this approach but imo the value it adds is pretty small to be prioritize rn